apps/functions-subscription/src/domain/trial.ts

```diff
@@ -15,12 +18,20 @@ interface TrialIdBrand {
   readonly TrialId: unique symbol;
 }
 export const TrialIdCodec = t.brand(
-  NonEmptyString,
-  (str): str is t.Branded<NonEmptyString, TrialIdBrand> => str.length > 0,
+  WithinRangeString(26, 27),
+  (str): str is t.Branded<WithinRangeString<26, 27>, TrialIdBrand> =>
+    str.length >= 26 && str.length < 27,
   'TrialId',
 );
 export type TrialId = t.TypeOf<typeof TrialIdCodec>;
 
+const ListTrialOptionsCodec = t.strict({
+  pageSize: t.number,
+  maximumId: t.union([TrialIdCodec, t.undefined]),
+  minimumId: t.union([TrialIdCodec, t.undefined]),
+});
```

Comment:
**suggestion**: improve this type

from a developer point of view, this type force to explicit the `maximumId` and the `minimumId`, even if the parameters can be `undefined`.

Make it possible to not explicit those parameters, when not needed

Reply:
Resolved in eff99bbaf41355bc1a98ee131ab9a4d2694136f4